src/Properties.ts

```diff
@@ -0,0 +1,39 @@
+import { Logger } from '@hmcts/nodejs-logging'
+import merge = require('lodash.merge')
```

Comment:
I think the syntax should be:
```suggestion
import { merge } from 'lodash.merge'
```

Reply:
Tried that but apparently the above is  the official way to do it.